homework_1/task_2/tax/progressiveTax.go

```diff
@@ -0,0 +1,95 @@
+package tax
+
+import (
+	"bufio"
+	"log"
+	"math"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+type TaxBracket struct {
+	LowerThreshold float64
+	UpperThreshold float64
+	Percentage float64
+}
+
+func CalculateTax(inputValue float64, taxBrackets []TaxBracket) (float64, error) {
```

Comment:
I could use `CalculateTax` directly (without ever calling `GetTaxBracketsFromFile`) which is good but there's no validation for the given `taxBrackets` in this function.